modules/transports/core/nhttp/src/main/java/org/apache/synapse/transport/passthru/TargetHandler.java

```diff
@@ -51,6 +51,7 @@
 import java.io.IOException;
 import java.util.Properties;
 
+import static org.apache.synapse.transport.passthru.TargetContext.CONNECTION_INFORMATION;
```

Comment:
It's better if we can remove static imports and use TargetContext.CONNECTION_INFORMATION in the code to get more readability.

Reply:
fixed with https://github.com/wso2/wso2-synapse/pull/1397/commits/32dd70c5631d28b5d28abc529d5e0bba9d4de498